src/images/google_cloud_sdk.rs

```diff
@@ -11,60 +10,65 @@ pub const DATASTORE_PORT: u16 = 8081;
 pub const FIRESTORE_PORT: u16 = 8080;
 pub const PUBSUB_PORT: u16 = 8085;
 
-const DEFAULT_DATASTORE_PROJECT: &str = "test";
-
-#[derive(Debug, Clone, Default)]
+#[derive(Debug, Clone)]
 pub struct CloudSdkArgs {
     pub host: String,
     pub port: u16,
     pub project: String,
-    pub emulator: Emulator,
+    pub emulator: Option<Emulator>,
+}
+
+impl Default for CloudSdkArgs {
+    fn default() -> Self {
+        Self {
+            host: HOST.to_owned(),
+            port: DEFAULT_PORT,
+            project: String::default(),
+            emulator: None,
+        }
+    }
 }
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq, Copy)]
 pub enum Emulator {
     Bigtable,
     Datastore,
     Firestore,
     PubSub,
-    Help,
```

Comment:
I did not like having this. By not having this, the image doesn't adhere to the req. that image args should have a meaningful default. And I think that's okay for images having many distinct services instead of just one. Just for the sake of some meaningful default, having such a not-so-meaningful (can also be considered as meaningless) did make much sense to me. WDYT?

Reply:
Image args should have a meaningful default if possible. But we should not implement `Default` if we cannot provide a meaningful one though.

In this case, I think we should remove the `Default` impl for `CloudSdkArgs`.

Reply:
Yeah, definitely. I thought the same. But wasn't sure whether we should separate `Args` from `Image`.